lib/fluent/plugin/out_file.rb

```diff
@@ -87,7 +102,13 @@ def configure(conf)
       @formatter = Plugin.new_formatter(@format)
       @formatter.configure(conf)
 
-      @buffer.symlink_path = @symlink_path if @symlink_path
+      if @symlink_path && @buffer.respond_to?(:path)
+        (class << @buffer; self; end).module_eval do
+          prepend SymlinkBufferMixin
```

Comment:
My concern is this plugin is also used in v0.12.
So if someone writes the patch to v0.12 out_file, v0.14 changes may conflict.